xbmc/utils/IBacklight.h

```diff
@@ -0,0 +1,13 @@
+#include "ServiceBroker.h"
+
+#include <memory>
+
+class IBacklight
+{
+public:
+  static void Register(std::shared_ptr<IBacklight> backlight) { CServiceBroker::RegisterBacklight(backlight); }
+
+  virtual bool SetBrightness(int brightness) = 0;
+  virtual int GetBrightness() = 0;
```

Comment:
While the implementation actually hasn't even a member for it, I'd still make this
`virtual int GetBrightness() const = 0;`
Same for other Getters.